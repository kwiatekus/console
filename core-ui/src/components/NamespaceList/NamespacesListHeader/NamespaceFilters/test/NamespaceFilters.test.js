import React from 'react';
import renderer from 'react-test-renderer';

import NamespaceFilters from './../NamespaceFilters';

describe('NamespaceFilters', () => {
  it('Renders with minimal props', () => {
    const component = renderer.create(
      <NamespaceFilters
        filters={[{ name: '1', name: '2' }]}
        updateFilters={() => {}}
      />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
