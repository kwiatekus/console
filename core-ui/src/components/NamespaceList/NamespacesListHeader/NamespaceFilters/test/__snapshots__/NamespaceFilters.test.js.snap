// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NamespaceFilters Renders with minimal props 1`] = `
<div
  className="fd-popover"
>
  <div
    onFocus={[Function]}
    onMouseDown={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="fd-popover__control"
    >
      <button
        className="fd-button--light sap-icon--filter"
        onClick={[Function]}
      />
    </div>
  </div>
</div>
`;
